substrate/client/rpc-spec-v2/src/chain_head/subscription/inner.rs

```diff
@@ -759,10 +759,13 @@ impl<Block: BlockT, BE: Backend<Block>> SubscriptionsInner<Block, BE> {
 			return Err(SubscriptionManagementError::SubscriptionAbsent)
 		};
 
+		// Keep only unique hashes.
+		let hashes = hashes.into_iter().collect::<HashSet<_>>();
```

Comment:
Maybe we should error on duplicated hashes instead?

Reply:
Yep that sounds good to me, I'll create a small PR in the spec to clarify this (part of https://github.com/paritytech/json-rpc-interface-spec/issues/133)